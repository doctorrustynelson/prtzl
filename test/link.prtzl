

Vertex a = <+ "a" +>;
Vertex b = <+ "b" +>;
Vertex c = <+ "c" +>;

link(a, b, 1);
link(a, c, 2);

bi_link(b, c, 3);

print_vertex(a);
print_vertex(b);
print_vertex(c);

