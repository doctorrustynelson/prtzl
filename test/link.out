a: 0 edge(s) in 2 edge(s) out
b: 2 edge(s) in 1 edge(s) out
c: 2 edge(s) in 1 edge(s) out
